src/libfetchers/git-utils.cc

```diff
@@ -151,11 +153,11 @@ struct GitRepoImpl : GitRepo, std::enable_shared_from_this<GitRepoImpl>
     {
         initLibGit2();
 
-        if (pathExists(path.native())) {
-            if (git_repository_open(Setter(repo), path.c_str()))
+        if (pathExists(path.string())) {
```

Comment:
I think the more we use `std::filesystem::path` the more it will go away --- for now I am fine with it because it has no effect on Unix and it gets Windows at least compiling.